internal/pkg/config/v1/config.go

```diff
@@ -217,6 +265,8 @@ func (c *Config) Save() error {
 	c.restoreOverwrittenContext(tempContext)
 	c.restoreOverwrittenAccount(tempAccount)
 	c.restoreOverwrittenKafka(tempKafka)
+	c.restoreOverwrittenAuthToken(tempAuthToken)
+	c.restoreOverwrittenAuthRefreshToken(tempAuthRefreshToken)
```

Comment:
Why do we need to do this? I've always been confused about this code.

Reply:
It's been encrypted to be saved. Restore them back to unencrypted version of tokens, so other commands / clients can use those tokens. Otherwise they'll just stay encrypted and can't be used anymore

Reply:
Why don't we just have a field for "AuthToken" and another field for "EncryptedAuthToken"?

Reply:
if you still have unencrypted `auth token` in your config, that'll be saved into config.json as well.

Reply:
You can set the serialization tag to `json:"-"` and it won't!

Reply:
i'll think about it and might do a separate pr, it'll also involve many changes in tests. For now i want to ship this change with today's release...

Reply:
If user is logged in using older cli, their config has token named `auth-token` that's unencrypted and i still want to read them?

Reply:
Oh true, this would be a breaking change. I'll add this to the v4 list.